netmanager/src/utils/setAuthToken.js

```diff
@@ -19,17 +20,18 @@ const setAuthToken = (token) => {
     }
 
     // Apply authorization token to every request if logged in
-    axios.defaults.headers.common['Authorization'] = token;
-    axios.interceptors.request.use((config) => {
-      const params = config.params || {};
+    // axios.defaults.headers.common['Authorization'] = token;
+    // axios.interceptors.request.use((config) => {
+    //   const params = config.params || {};
 
-      return { ...config, params: { tenant, ...params } };
-    });
-  } else {
-    // Update auth header token to support data access for non-auth users
-    axios.defaults.headers.common[
-      'Authorization'
-    ] = `JWT ${process.env.REACT_APP_AUTHORIZATION_TOKEN}`;
+    //   return { ...config, params: { tenant, ...params } };
+    // });
   }
+  // else {
+  //   // Update auth header token to support data access for non-auth users
```

Comment:
Hi @OchiengPaul442 , could we minimise commented out code? Unless we plan to use it in the very immediate future?